torchgeo/datasets/zuericrop.py

```diff
@@ -230,3 +246,85 @@ def _download(self) -> None:
                     filename=filename,
                     md5=md5 if self.checksum else None,
                 )
+
+    def _validate_bands(self, bands: Tuple[str, ...]) -> None:
+        """Validate list of bands.
+
+        Args:
+            bands: user-provided tuple of bands to load
+        Raises:
+            AssertionError: if ``bands`` is not a tuple
+            ValueError: if an invalid band name is provided
+
+        .. versionadded:: 0.2
+        """
+        assert isinstance(bands, tuple), "The list of bands must be a tuple"
+        for band in bands:
+            if band not in self.band_names:
+                raise ValueError(f"'{band}' is an invalid band name.")
+
+    def plot(
+        self,
+        sample: Dict[str, Tensor],
+        time_step: int = 0,
+        show_titles: bool = True,
+        suptitle: Optional[str] = None,
+    ) -> plt.Figure:
+        """Plot a sample from the dataset.
+
+        Args:
+            sample: a sample return by :meth:`__getitem__`
+            time_step: time step at which to access image, beginning with 0
+            show_titles: flag indicating whether to show titles above each panel
+            suptitle: optional suptitle to use for figure
+
+        Returns;
+            a matplotlib Figure with the rendered sample
+
+        .. versionadded:: 0.2
+        """
+        rgb_indices = []
+        for band in self.RGB_BANDS:
+            if band in self.bands:
+                rgb_indices.append(self.bands.index(band))
+            else:
+                raise ValueError("Dataset doesn't contain some of the RGB bands")
+
+        ncols = 2
+        image, _ = sample["image"][time_step, rgb_indices, ...], sample["mask"]
+        _, boxes = sample["label"], sample["boxes"]
+
+        boxes = draw_bounding_boxes(
+            image=image.to(torch.uint8),  # type: ignore[attr-defined]
+            boxes=sample["boxes"],
```

Comment:
It seems like the `boxes` variable you defined above isn't actually used anywhere

Reply:
i was not sure whether to plot boxes, so I was gonna change the plotting to whatever is desired.